plugins/azure-sites-backend/src/service/router.ts

```diff
@@ -52,7 +65,34 @@ export async function createRouter(
     '/:subscription/:resourceGroup/:name/start',
     async (request, response) => {
       const { subscription, resourceGroup, name } = request.params;
-      console.log('starting...');
+      const token = getBearerTokenFromAuthorizationHeader(
+        request.header('authorization'),
+      );
+      const entity = request.body.entity;
+      const resourceRef = stringifyEntityRef(entity);
+      const matchAnnotation = name.match(
+        new RegExp(
```

Comment:
continuing the discussion about https://github.com/backstage/backstage/pull/16171#discussion_r1106414980 :
how can you be sure that this RegExp matches the logic in https://github.com/backstage/backstage/blob/8761094ac87a78a36168be0976ddba38f6dbed46/plugins/azure-sites-backend/src/api/AzureSitesApi.ts#L77`?
Isn't better to retrieve here all the items and check that an item with `name` exists in the list?

Reply:
for example, if an entity of name `awesomewebsite` exists and I pass `awesomewebsiteawesomewebsite` as name to this endpoint, the endpoint won't throw any error

Reply:
I updated the logic, removed `RegEx` and matching the site by one on one